js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -526,6 +526,112 @@ module.exports = {
              * @name GridBaseColumn.formItem
              * @type dxFormSimpleItem
              */
+            /**
+             * @name dxDataGridColumn.type
+             * @publicName type
+             * @type Enums.GridCommandColumnType
+             */
+            /**
+             * @name dxTreeListColumn.type
+             * @publicName type
+             * @type Enums.TreeListCommandColumnType
+             */
+            /**
+             * @name dxDataGridColumn.buttons
+             * @type Array<Enums.GridColumnButtonName,dxDataGridColumnButton>
+             * @default undefined
```

Comment:
`@default` is currently used by the Site only, and setting it to `undefined` makes no sense (except for overriding an inherited value)

@viterobk, @RomanTsukanov

Reply:
"default: undefined" gives a user more information than nothing.